changelog/17201.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Similar to my comment in https://github.com/hashicorp/vault/pull/17199/files#r974485555, I don't think we need this CL entry if nothing changed in the plugin itself since its last release.